File: examples/ex011_largura_altura_parede_pintar/ex011.js
function calculate() {

    let inputHeight = document.querySelector('#input1').value;
    let inputHeightNumber = Number(inputHeight);
    let inputWidth = document.querySelector('#input2').value;
    let inputWidthNumber = Number(inputWidth);
    let wallArea = inputHeightNumber * inputWidthNumber;
    let divAnswer = document.querySelector('.answer');

    if (inputHeight == "" || inputWidth == "") {

        divAnswer.innerHTML = `<div class="answer error">Informe os dois valores!</div>`;

    } else {

        divAnswer.innerHTML = "";
        divAnswer.innerHTML += `<div class="answer">Sua parede com <strong>${inputHeightNumber.toFixed(1)}m</strong> de largura e <strong>${inputWidthNumber.toFixed(1)}m</strong> de altura tem uma área de <strong>${wallArea.toFixed(1)}m²</strong>.</div>`;
        divAnswer.innerHTML += `<div class="answer">Ou seja, serão necessários <strong>${(wallArea/2).toFixed(1)} litros</strong> de tinta para pintar a sua parede!</div>`;
    }
}

File: python/world_1/examples/2_tratando_dados_fazendo_contas/ex011_largura_altura_parede_pintar/ex011.py
# Faça um programa que leia a largura e a altura de uma parede em metros, calcule a sua área e a quantidade de tinta necessária para pintá-la, sabendo que cada litro de tinta pinta uma área de 2m².

wallHeight = float(input('Qual é a altura da parede? '))
wallWidth = float(input('Qual é a largura da parede? '))
wallArea = wallHeight * wallWidth

print(f'A parede a ser pintada com {wallHeight:.1f} de altura e {wallWidth:.1f} de largura tem {wallArea:.1f}m². \nOu seja, serão necessários {(wallArea/2):.1f} litros de tinta para pintar a área toda!')
